src/ResourceManager/Scheduler/Commands.Scheduler/packages.config

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Scheduler" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
+  <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
+  <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
+  <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.0" targetFramework="net45" />
```

Comment:
When I increment it to 2.28.1, i'm getting Assembly mismatch. I believe its due to Commands.ResourceManager.Common still referring to 2.28.0.